Projects/Feature/HomeFeature/Sources/Scene/HomeViewController.swift

```diff
@@ -140,6 +143,30 @@ final class HomeViewController: BaseStoredViewController<HomeStore> {
         let sharedState = store.state.share()
             .receive(on: DispatchQueue.main)
 
+        sharedState
+            .map(\.profileImageUrl)
+            .sink(with: self, receiveValue: { owner, profileImageURL in
+                guard let profileImageURL,
+                      let imageURL = URL(string: profileImageURL)
+                else {
+                    owner.myInfoImageView.image = .Dotori.personCircle
+                    return
+                }
+
+                let request = ImageRequest(
+                    url: URL(string: profileImageURL),
```

Comment:
위에서 optional unwrapping했으니 그거 쓰는거 어떤가요
```suggestion
                    url: imageURL,
```

Reply:
> 위에서 optional unwrapping했으니 그거 쓰는거 어떤가요

e4d0d83028e2991c65b066b326db574812214dde 수정했습니당